docs/pages/hdmi_cec_driver_L2_Low-Level_TestSpecification.md

```diff
@@ -0,0 +1,309 @@
+# HDMI CEC DRIVER L2 Low Level Test Specification and Procedure Documentation
+ 
+## Table of Contents
+            
+- [HDMI CEC DRIVER L2 Low Level Test Specification and Procedure Documentation](#hdmi-cec-driver-l2-low-level-test-specification-and-procedure-documentation)
+
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+    - [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+    - [Definitions](#definitions)
+    - [References](#references)
+  - [Level 2 Test Procedure](#level-2-test-procedure)
+ 
+## Overview
+
+This document describes the level 2 testing suite for the HDMI CEC DRIVER module.
+ 
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
```

Comment:
need to add `DUT`  Device Under Test
remove `SOC` is not used

Reply:
De-prioritizing this to handle other priorities like UTLogger2.0, config framework,sink/source distinction etc